### PR TITLE
move test_z1

### PR DESCRIPTION
**Pull request by nstarman**
Signed-off-by: Nathaniel Starkman (@nstarman) <nstarkman@protonmail.com>

Move another test to new test suite.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: astropy/cosmology/tests/test_flrw.py [hunk 1]
```diff
@@ -737,7 +737,7 @@ def test_w_fail(self, cosmo, z, exc):
 
     # -------------------------------------------
 
-    @pytest.mark.skipif(HAS_SCIPY, reason="scipy is not installed")
+    @pytest.mark.skipif(not HAS_SCIPY, reason="scipy is not installed")
```
Comment (nstarman) [inline astropy/cosmology/tests/test_flrw.py:740]: lol. If it passes this way it means it should be removed.

Comment (nstarman) [reply astropy/cosmology/tests/test_flrw.py:740]: Done.

**Review by mhvk — COMMENTED** [review]
Hmm, are you sure about this? This really feels like a stand-alone overall systems test, not something that should also be run on anything that subclasses `TestFlatLambdaCDM`. I'd even tend to keep it in `test_cosmology`.

**Comment by nstarman** [comment]
Never thought of keeping ``test_cosmology.py`` around. Not a bad idea! There are definitely a number of tests I was wondering how to incorporate. I'll retool this to start transitioning ``test_cosmology.py`` into stand-alone overall systems tests. Thx.

File: astropy/cosmology/tests/test_cosmology.py [hunk 2]
```diff
@@ -9,12 +11,42 @@
...
+    assert allclose(cosmo.luminosity_distance(1),
+                    [6729.2, 6729.6, 6729.5976] * u.Mpc, rtol=1e-4)
+    assert allclose(cosmo.lookback_time(z),
+                    [7.841, 7.84178, 7.843] * u.Gyr, rtol=1e-3)
+    assert allclose(cosmo.lookback_distance(1),
+                    [2404.0, 2404.24, 2404.4] * u.Mpc, rtol=1e-3)
 
 
+###############################################################################
+# TODO:
```
Comment (mhvk) [inline astropy/cosmology/tests/test_cosmology.py:48]: Is the TODO that these more specific ones are still moved elsewhere?

**Review by mhvk — APPROVED** [review]
Looks fine, but do finish the `TODO`!

**Comment by mhvk** [comment]
p.s. Failures are real!

File: astropy/cosmology/tests/test_cosmology.py (hunk 2 above)
Comment (nstarman) [reply astropy/cosmology/tests/test_cosmology.py]: Yes. All the following tests are on my todo list to process.

Comment (nstarman) [reply astropy/cosmology/tests/test_cosmology.py]: I added a comment to this effect.

**Comment by nstarman** [comment]
Rebased now that numpy / windows CI issues are fixed.

**Pull request merged**